tensorflow/c/kernels.cc

```diff
@@ -289,19 +289,24 @@ TF_Tensor* TF_AllocateTemp(TF_OpKernelContext* context, TF_DataType dtype,
   TF_SetStatus(status, TF_OK, ""); 
   tensorflow::gtl::ArraySlice<tensorflow::int64> dimarray(
       reinterpret_cast<tensorflow::int64*>(dims), num_dims);
+  if (!attributes->struct_size) {
```

Comment:
This would fail if `attributes` is a nullptr.
Change to:
`if (attributes && !attributes->struct_size) {`

Also, add space in the error message below after "TF_AllocatorAttributes struct" otherwise it will be "TF_AllocatorAttributes structsize member..." when printed